<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>386</width>
    <height>269</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QLabel" name="label_title">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Title</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_title">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>40</y>
     <width>291</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_duration">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>120</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Duration</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_durationHours">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>120</y>
     <width>42</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_durationMinutes">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>120</y>
     <width>42</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_durationHours">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>120</y>
     <width>16</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>H</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_durationMinutes">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>120</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>M</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_writeToFile">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>230</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Write To File</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_director">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Director</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_director">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>80</y>
     <width>291</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_releaseDate">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>160</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Release Date</string>
   </property>
  </widget>
  <widget class="QDateEdit" name="dateEdit_releaseDate">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>160</y>
     <width>101</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
